kythe/javatests/com/google/devtools/kythe/analyzers/java/testdata/pkg/Deprecation.java

```diff
@@ -1,25 +1,28 @@
 package pkg;
 
-//- @+8Deprecation defines/binding DeprecationClass
+//- @+9Deprecation defines/binding DeprecationClass
 
 /**
  * This the javadoc summary fragment, not the {@code @deprecated} tag.
  *
  * @deprecated this class is obsolete; prefer {@link Comments}
  * @author this is an author tag, not the {@code @deprecated} tag.
  */
+@Deprecated
```

Comment:
I thought this file was to test deprecated javadoc in the absence of the annotation?

Reply:
I wanted to confirm that we save the text from the javadoc if both the javadoc and the annotation are present.

The `deprecatedFieldEmpty` example below still exercises the case where there's javadoc but no annotation.

So I think this has OK coverage of the current logic, but I can split this out into more test cases if you'd prefer?

Reply:
Yes to preserving the javadoc deprecation text if both are present.

Test coverage LGTM.